src/DependencyInjection/CmfCoreExtension.php

```diff
@@ -318,16 +319,31 @@ private function loadPublishWorkflow($config, XmlFileLoader $loader, ContainerBu
         $container->setParameter($this->getAlias().'.publish_workflow.view_non_published_role', $config['view_non_published_role']);
         $loader->load('publish-workflow.xml');
```

Comment:
shouldn't this line go somewhere behind the checks? Or is the removing stuff enough?

I still got
```
!!    The service "cmf_core.publish_workflow.checker.default" has a dependency on  
!!  
!!  
!!     a non-existent service "security.token_storage".   
````
when installing your branch.

```
maximilian@maximilian-ThinkPad-T540p:~/OpenSource/Cmf/install-testing/build/symfony-cmf/core-bundle/dev-routing-bundle-optional/test-app$ composer show symfony-cmf/*
symfony-cmf/core-bundle dev-routing-bundle-optional 75fbb35 Symfony CMF Core Bundle
``